sbin/root-mount

```diff
@@ -3,6 +3,7 @@
 # Jolla btrfs root mounting script, adapted for ext4
 #
 # Copyright (C) 2014 Jolla Ltd.
+# Copyright (c) 2021. Open Mobile Platform LLC.
```

Comment:
No full-stop after the date, see e.g. https://github.com/sailfishos/sailjail/blob/master/daemon/test/test_appinfo.c#L2
```suggestion
# Copyright (c) 2021 Open Mobile Platform LLC.
```

Reply:
Thanks